### DISCUSSION TITLE
drill to detail - disable/limit columns for dedicated charts?

### DISCUSSION THREAD

**Discussion by nigzak** [apache/superset#25655] [General]
I have dedicated charts in superset which are based on high aggregated data to avoid high peak loads for superset which would result in HOURS waiting for response (too many data).

I have a logic which pre-calculates as example daily or weekly data to show bars (as example GOOD count vs. BAD count).
If I have DRILL-TO-DETAIL enable on this chart, the only thing which the user sees is the aggregated stored data - which does not help.

Might it not be good to have a flag on a chart to enable/disable a drill-to-detail?
Or as alternative to show OTHER data in the drill-to-detail (like instead show the raw data with a limit or else)?

Also probably nice would be in the drill-to-detail to have possibility to setup which columns are shown in my opinion.
Example: You have data inside the table which is pre-calculated and not understandable for the user - probably even not used for the charts. It would be probably a good option to let the user select which columns are used for a drilldown (and if NO column is selected this might be the option to DISABLE it for the dedicated chart)

Default could be "all" is selected and it is working like today with options to remove ("x") columns for the drilldown so a possible migration to this functionality would not kill anything as of now.

**Comment by nigzak** [comment]
seems almost nobody is interested in this topic -> closing

**Comment by shingote15** [comment]
Interested
